pulsar-broker/src/main/java/org/apache/pulsar/broker/transaction/buffer/impl/TransactionBufferHandlerImpl.java

```diff
@@ -288,12 +260,16 @@ private void checkPendingRequests() {
     }
 
     private void invalidateLookupCache(OpRequestSend op) {
+        if (lookupCache.get(op.topic) == op.cnx) {
+            safeInvalidateCache(op.topic);
+        }
+    }
+
+    private void safeInvalidateCache(String topicName) {
         try {
-            if (lookupCache.get(op.topic) == op.cnx) {
-                lookupCache.invalidate(op.topic);
-            }
-        } catch (ExecutionException e) {
-            lookupCache.invalidate(op.topic);
+            lookupCache.synchronous().invalidate(topicName);
+        } catch (NullPointerException ignore) {
+            // no-op
```

Comment:
Why NPE?

Reply:
ah... It's a mistake, fixed. 

thanks for your review.